.eslintrc.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
## Enforce Gitflow workflow
- [ ] Please ensure that you get to work with `Git Flow: production master branch + main develop branch + Pull Requests` from `feature branches` to the `develop` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/section/git_flow_requirement.png) from Pathwright). You are currently opening a `Pull Request` from a `feature branch` to the `main` branch instead of the `development` branch. Kindly enforce this change.